config/config.yaml

```diff
@@ -41,3 +42,6 @@ get_padded_baits:
   target_regions: "data/genomic_regions/exons_variants_pharmacogenomics_18_06_2019_ex_cyp2d6.bed"
   padding: 100
   file_format: "list"
+
+depth_of_baits:
+  container: "docker://hydragenetics/gatk4:4.2.2.0"
```

Comment:
Different GATK4 version than previously. Better to use the same if possible.

Reply:
Right, I fixed both the config files to have the same version.